node/lib/util/synthetic_gc_util.js

```diff
@@ -68,6 +68,8 @@ let detail = {
         if (path.endsWith("-git")) {
             path = path.replace(new RegExp("-git$"), ".git");
         }
+
+        return path;
```

Comment:
Did the tests pass before this fix? If so, do we need to change them?

Reply:
Yeah, tests were using absolute paths for submodules. So this code was not hit, added test with relative paths. Thanks for catching.